src/components/content/order/locks/Locks.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useRef, useState } from 'react';
+import { Button, Form, Space, Switch } from 'antd';
+import '../../../../styles/locks.css';
+import {
+    ApiError,
+    DeployedServiceDetails,
+    Response,
+    ServiceLockConfig,
+    ServiceService,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../xpanse-api/generated';
+import { useMutation } from '@tanstack/react-query';
+
+export const Locks = ({
+    currentSelectedService,
+}: {
+    currentSelectedService: DeployedServiceDetails | VendorHostedDeployedServiceDetails;
+}): React.JSX.Element => {
+    const [form] = Form.useForm();
+    const requestResult = useRef<string[]>([]);
+    const serviceLockConfig = useRef<{ id: string; lockConfig: ServiceLockConfig } | undefined>(undefined);
+    const [destroyChecked, setDestroyChecked] = useState<boolean>(
+        currentSelectedService.lockConfig?.destroyLocked ? currentSelectedService.lockConfig.destroyLocked : false
+    );
+    const [modifyChecked, setModifyChecked] = useState<boolean>(
+        currentSelectedService.lockConfig?.modifyLocked ? currentSelectedService.lockConfig.modifyLocked : false
+    );
+
+    const lockRequest = useMutation({
+        mutationFn: (requestBody: { id: string; lockConfig: ServiceLockConfig }) => {
+            return ServiceService.changeServiceLockConfig(requestBody.id, requestBody.lockConfig);
+        },
+        onSuccess: () => {
```

Comment:
no alert displayed. it is not clear whether the request is successful or no. 
We must show a small alert.